calendar-bundle/src/Resources/contao/classes/Calendar.php

```diff
@@ -373,6 +373,11 @@ public function getSearchablePages($arrPages, $intRoot=0, $blnIsSitemap=false)
 				{
 					while ($objEvents->next())
 					{
+						if ($blnIsSitemap && $objEvents->robots === 'noindex,nofollow')
```

Comment:
should this check the page settings as well?

Reply:
It's already checked above, so no need to check it again

https://github.com/contao/contao/blob/449a6fbb3717af62b5f40f1f71cd9f5c71fb87f1/calendar-bundle/src/Resources/contao/classes/Calendar.php#L357